src/app/bisoncoin/src/components/Button.vue

```diff
@@ -10,12 +12,15 @@ import { Component, Prop, Vue } from "vue-property-decorator";
 @Component
 export default class Button extends Vue {
   @Prop() private label!: string;
+  @Prop() private dest!: string;
 }
 </script>
 
-<style>
+<style lang="scss">
+@import "../style.scss";
+
 .button {
-  background-color: #00a3e050;
+  background-color: $button-background-color;
   width: 100px;
   height: 50px;
   border-radius: 25px;
```

Comment:
We can move the box-shadow information to a variable in style.scss because I think we will reuse it. Also, it is really long and I think it will be cleaner in a variable.

Reply:
True, true